de.wwu.md2.framework/src/de/wwu/md2/framework/generator/mapapps/AppClass.xtend

```diff
@@ -60,6 +63,12 @@ class AppClass {
     
     // bundles section used to overwrite or add any property defined by components in the manifest.json of bundles
     "bundles": {
+        "md2_workflow_store": {
+            "MD2WorkflowStore": {
+                    "uri": "«dataContainer.main.defaultConnection.uri»workflowState/",
```

Comment:
Added a workflowManager connection and wrote Util methods for building the uris for workflowStateWS and EventHandlerWS